### ISSUE TITLE
Image Plugin Not Rendering with Django 1.8

### ISSUE THREAD

**Issue by esomething** [divio/cmsplugin-filer#190]
I'm using Django 1.8 and have found that the image template at cmsplugin_filer_image/plugins/image/default.html is not rendering.

I know that Django 1.8 has a number of changes to the template system, including one to the behavior of select_template. Instead of returning an object of type `django.template.base.Template` it returns and object of type `django.template.backends.django.Template`.

Thus, on line 104, this code:

```
    def render(self, context, instance, placeholder):
        self.render_template = select_template((
            'cmsplugin_filer_image/plugins/image.html',  # backwards compatibility. deprecated!
            self.TEMPLATE_NAME % instance.style,
            self.TEMPLATE_NAME % 'default')
        )
```
returns a `django.template.backends.django.Template` object with 1.8 and a `django.template.base.Template` with < 1.8.

I've found that adding a line after this:

```
self.render_template = self.render_template.template
```

will make the plugin render, but I'm not sure it's the correct solution.

**Comment by sephii** [comment]
This is caused by https://github.com/divio/django-cms/blob/3.1.2/cms/plugin_rendering.py#L63:

```
    if not processors:
        processors = []
    if isinstance(template, six.string_types):
        content = render_to_string(template, context_instance=context)
    elif isinstance(template, Template):
        content = template.render(context)
    else:
        content = ''
```

I guess the best solution would be either to change the code in cmsplugin-filer as you proposed, or to replace the code in django-cms by:

```python
from django.template.backends.django.Template as BackendTemplate
# ...
elif isinstance(template, (Template, BackendTemplate)):
    content = template.render(context)
```
I'm not comfortable enough with Django 1.8 changes to say which solution would be better.

**Comment by yakky** [comment]
This has been fixed in 3.1.x branch (https://github.com/divio/django-cms/blob/support/3.1.x/cms/plugin_rendering.py#L64) which is due to release as 3.1.3 shortly

**Comment by czpython** [comment]
Fixed on CMS 3.1.3
